project-resources/community-health-files-guide.md

```diff
@@ -0,0 +1,61 @@
+# Guide to Community Health Files for OpenJS Foundation Projects
+
+Community health files are documents related to project collaboration. Most commonly, these include:
+
+- `CODE_OF_CONDUCT.md`
+- `CONTRIBUTING.md`
+- `GOVERNANCE.md`
+- `SECURITY.md`
```

Comment:
We also want to mention CHARTER.md here and say where it should go (depends on whether it's a project with a single repo (in which case it goes in the project) or a project with multiple repos, in which case it must go in a dedicated repo (which ideally should have a common name).

Reply:
`CHARTER.md` is not a supported file type, so we'll need to add a caveat

Reply:
Right, but _we_ care about projects having such a file and having it in a place that's easy for us to reference.